lfs.c

```diff
@@ -2548,7 +2548,7 @@ static int lfs_file_flush(lfs_t *lfs, lfs_file_t *file) {
                 }
             }
         } else {
-            file->ctz.size = lfs_max(file->pos, file->ctz.size);
+            file->pos = lfs_max(file->pos, file->ctz.size);
```

Comment:
@geky This line is either incorrect or useless, because `file->pos` is overwritten by `pos` on line 2560.  Should it be assigning to `pos`?

Reply:
Probably incorrect, since in what's below `file->pos` is used to replace `file.ctz.size`.

Reply:
Hi @pabigot, thanks for raising this. To double check is this fixed in https://github.com/ARMmbed/littlefs/pull/257?

Reply:
I didn't touch this material in #257; that fixes a different bug.  I misunderstood what this patch was doing the first couple time I read it.  As far as I know there's nothing wrong left here after this was merged.

Reply:
Ah ok, good to know